Yoast/wordpress-seo
# Improves plugin icon consistency

## Summary
* Replaced the old plugin icons with the new designs.


This PR can be summarized in the following changelog entry:

* Improves the consistency of the plugin icons.

## Relevant technical choices:

* Replaced the old plugin icons with the new designs.

## Test instructions

This PR can be tested by following these steps:
* Go to the SEO tab in your local environment, and check if all icons are displayed.
  * Go to General, check the ads in the right sidebar.
  * `Yoast_SEO_Icon.svg`:
    * Configuration wizard.
    * Dashboard: the loading icon of the `Yoast SEO Posts Overview` (refresh the page and quickly scroll down, or edit the html).
    * Yoast dashboard: on a badge. Seems this does not exist anymore. Should this CSS be removed?
    * Notifications: trash post, delete post (don't know how to trigger this, but the code is the same as the other 2 notifications) and trash taxonomy.
    * Dashboard -> Help center -> Get support.
 

## UI changes

## Documentation

## Quality assurance


Fixes #10360
